official/benchmark/models/resnet_imagenet_main.py

```diff
@@ -37,6 +38,40 @@
 from official.vision.image_classification.resnet import resnet_model
 
 
+def cluster_last_three_conv2d_layers(model):
+  last_three_conv2d_layers =  [
+      layer for layer in model.layers
+      if isinstance(layer, tf.keras.layers.Conv2D) and
+      not isinstance(layer, tf.keras.layers.DepthwiseConv2D)
+    ]
+  last_three_conv2d_layers = last_three_conv2d_layers[-3:]
+
+  cluster_weights = tfmot.clustering.keras.cluster_weights
+  CentroidInitialization = tfmot.clustering.keras.CentroidInitialization
+  clustering_params1 = {
```

Comment:
In my other comment, I suggested just directly inlining clustering_params1 and clustering_params2 with the call to `cluster_weights`. Was there a reason not to do so?

Inlining removes a layer of indirection.